TestCase/Managers/GameManager.cs

```diff
@@ -0,0 +1,100 @@
+﻿namespace TestCase.Managers
+{
+    public class GameManager
+    {
+        private readonly BoardManager _boardManager;
+        private readonly InputManager _inputManager;
+
+        private bool isPlayerXTurn = true;
+        private bool isPlayerOTurn = false;
+
+        private int turnCounter = 0;
+
+        public GameManager()
+        {
+            this._boardManager = new BoardManager();
+            this._inputManager = new InputManager();
+        }
+
+        public void InputFromUser()
+        {
+            if (isPlayerXTurn)
```

Comment:
```suggestion
            Console.WriteLine($"\nit is currently Player {isPlayerXTurn ? GlobalConsts.X_PLAYER_TOKEN : GlobalConsts.O_PLAYER_TOKEN}'s turn");
```
Can use string interpolation and the ternary what character to write. The rest of the text is exactly the same, so it'll avoid repeated code :)